Framework/Source/Framework.h

```diff
@@ -31,6 +31,7 @@
 #include <memory>
 #include "glm/glm.hpp"
 #include <iostream>
+#include <string>
```

Comment:
Why is here? Framework.h is included by every file in the solution, we should try to minimize the amount of headers we put here.
Move it to whichever file that actually requires it.